packages/client/nft-earning/frontend/src/App.js

```diff
@@ -0,0 +1,319 @@
+    await nftService.askContractToMintNft(currentAccount);
+    setMining(false);
+  }
+
+  const checkIfWalletIsConnected = async () => {
+    const { ethereum } = window;
+
+    if (!ethereum) {
+      console.log("Make sure you have metamask!");
+      return;
+    } else {
+      console.log('asdf')
+    }
+
+    const accounts = await ethereum.request({ method: 'eth_accounts' });
+
+    if (accounts.length !== 0) {
+      const account = accounts[0];
+      setCurrentAccount(account);
+      nftService.setupEventListener(listenerCallback);
+    } else {
+      console.log("No authorized account found");
+    }
+  }
+
+  useEffect(() => {
+    const fetchNumNFTsMinted = async () => {
+      const num = await nftService.getNumMinted();
+      setNumMinted(num);
+    };
+    void fetchNumNFTsMinted();
+  }, [currentAccount])
+
+  useEffect(() => {
+    checkIfWalletIsConnected();
+  }, [])
+
+
+  return (
+    <div className="App">
+      <Navbar className="justify-content-between" variant="dark">
+        <img src={logo} width={50} height={50} className="logo"  />
+        <div>
+          <Chain chainId={currentChainID} />{' '}
+          <button className="cta-button connect-wallet-button" style={{height: "38px"}} disabled={isLogged} onClick={SignIn} variant="primary">{isLogged ? shortAddr() : "Connect"}</button>{' '}
+          <Button onClick={SignOut} style={{visibility: isLogged ? "visible" : "hidden"}} variant="danger">X</Button>
+        </div>
+      </Navbar>
+
+      <div className="message-list" >
+        {
+          messages.map((item,i) => (
+              <Message head={item.head} body={item.body} variant={item.variant} id={i} key={i} />
+          ))
+        }
+      </div>
+
+      <div className="container">
+        <div className="header-container">
+          <p className="header gradient-text">Goldfinch Flight Academy</p>
+          <p className="sub-text">
+            Earn your NFT
+          </p>
+            <button onClick={handleClickMint} style={{visibility: isLogged ? "visible" : "hidden"}} className="cta-button connect-wallet-button">
+              Mint NFT
+            </button>
+          <div style={{height: 100}} />
+          {mining && <img src={miningGif} width={100} height={100}/>}
+        </div>
+        {svg && (
+          <>
+            <p className="sub-text">Your self improover:</p>
+            <a
+              className="footer-text"
+              href={url}
+              target="_blank"
+              rel="noreferrer"
+            >View on Opensea</a>
+            <img src={`data:image/svg+xml;base64,${svg}`} height={300} width={300}/>
+          </>
+        )}
+
+        <div className="footer-container">
+          <img alt="Twitter Logo" className="twitter-logo" src={twitterLogo} />
+          <a
+            className="footer-text"
+            href={TWITTER_LINK}
+            target="_blank"
+            rel="noreferrer"
+          >{`built by @${TWITTER_HANDLE}`}</a>
```

Comment:
can you update this to say `built by the @${TWITTER_HANDLE} community`?

Reply:
Thank you, done!